source/appModules/foobar2000.py

```diff
@@ -46,6 +46,8 @@ class AppModule(appModuleHandler.AppModule):
 
 	def _get_statusBar(self):
 		# #11082: retrieve status bar handle and resulting NVDA object from playlist window.
+		# If only default status bar routine is used to assign status bar object,

```

Comment:
Do you mean that calling `api.getStatusBar()` from here leads to a recursion error? That's true, and was a concern when reviewing #9792. In the end I decided that any future work could split that function to break the recursion.

This is really a tangent to what I was trying to ask. The way that this works is different from `api.getStatusBar()`, is there a reason? In `api.getStatusBar()` we hit test 1 pixel up from the bottom of the foreground object and walk up the tree looking for the statusbar role. This implementation enumerates all child windows of the ForegroundObject looking for this particular class name.